Shared/Services/Time.swift

```diff
@@ -15,4 +15,17 @@ public class Time {
         
         return dateFormatter.date(from: dateString)
     }
+
+    public static func stringDateFromISOString(_ dateString: String) -> String {
```

Comment:
since this function is called `stringDateFromISOString` does that mean that I can generate an string like `Feb. 5th 2024` with it?